### PR TITLE
DOMA-1185 Make client and backend role to be required

### PR DESCRIPTION
**Pull request by MrFoxPro**

### CODE REVIEW

File: apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const InviteNewOrganizationEmployeeService = new GQLCustomSchema('InviteNewOrgan
     types: [
         {
             access: true,
-            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String, phone: String!, name: String, role: OrganizationEmployeeWhereUniqueInput, position: String, specializations: [TicketCategoryClassifierWhereInput!]}',
+            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String, phone: String!, name: String, role: OrganizationEmployeeWhereUniqueInput!, position: String, specializations: [TicketCategoryClassifierWhereInput!]}',
```
Comment (dkoviazin) [inline apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js:21]: Why role has OrganizationEmployeeWhereUniqueInput and not OrganizationEmployeeRoleWhereUniqueInput?

Comment (MrFoxPro) [reply apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js:21]: fixed!

File: apps/condo/domains/organization/components/CreateEmployeeForm.tsx [hunk 2]
```diff
@@ -149,8 +154,6 @@ export const CreateEmployeeForm: React.FC = () => {
                                                 <Col span={24}>
                                                     <Form.Item name={'role'} label={RoleLabel} {...INPUT_LAYOUT_PROPS} labelAlign={'left'} >
```
Comment (AntonAL) [inline apps/condo/domains/organization/components/CreateEmployeeForm.tsx:155]:
```suggestion
                                                    <Form.Item name={'role'} label={RoleLabel} required {...INPUT_LAYOUT_PROPS} labelAlign={'left'} >
```

File: apps/condo/domains/organization/components/EmployeeProfileForm/index.tsx [hunk 3]
```diff
@@ -106,7 +108,7 @@ export const EmployeeProfileForm = () => {
                                         name={'role'}
                                         label={RoleLabel}
```
Comment (AntonAL) [inline apps/condo/domains/organization/components/EmployeeProfileForm/index.tsx:109]:
```suggestion
                                        label={RoleLabel}
                                        required
```

File: apps/condo/domains/organization/utils/testSchema/Organization.js [hunk 4]
```diff
@@ -47,14 +48,20 @@ async function registerNewOrganization (client, extraAttrs = {}, { raw = false }
     })
     if (raw) return { data, errors }
     expect(errors).toEqual(undefined)
-    return [data.obj, attrs]
+    const { data: { objs: roles } } = await client.query(OrganizationEmployeeRole.GET_ALL_OBJS_QUERY, {
+        data: {
+            organization: { id: data.obj.id },
+        }
+    })
+    return [data.obj, attrs, roles]
```
Comment (AntonAL) [inline apps/condo/domains/organization/utils/testSchema/Organization.js:56]: Please, use `OrganizationEmployeeRole` from `apps/condo/domains/organization/utils/testSchema`.

**Pull request closed without merging**